### PR TITLE
section commit A for July 8th, '20

### PR DESCRIPTION
**Pull request by croqueGrec09**
elimination of PrivateProperty abstract class, move of PersonPrivateProperty to PersonProperty

### CODE REVIEW

File: app/src/groovy/de/laser/api/v0/ApiWriterHelper.groovy [hunk 1]
```diff
@@ -97,7 +97,7 @@ class ApiWriterHelper {
             person.contacts  = getContacts(it.contacts, null, person)
 
             def properties = getProperties(it.properties, person, contextOrg)
-            person.privateProperties = properties['private']
+            person.customProperties = properties['private']
```
Comment (Moetez16) [inline app/src/groovy/de/laser/api/v0/ApiWriterHelper.groovy:100]: person.privateProperties != person.customProperties

File: app/src/groovy/de/laser/api/v0/ApiMapReader.groovy [hunk 2]
```diff
@@ -28,7 +28,7 @@ class ApiMapReader {
             // References
             result.contacts     = ApiCollectionReader.getContactCollection(prs.contacts, allowedContactTypes) // com.k_int.kbplus.Contact
             result.addresses    = ApiCollectionReader.getAddressCollection(prs.addresses, allowedAddressTypes) // com.k_int.kbplus.Address
-            result.properties   = ApiCollectionReader.getPrivatePropertyCollection(prs.privateProperties, context) // com.k_int.kbplus.PersonPrivateProperty
+            result.properties   = ApiCollectionReader.getPrivatePropertyCollection(prs.customProperties, context) // com.k_int.kbplus.PersonPrivateProperty
```
Comment (Moetez16) [inline app/src/groovy/de/laser/api/v0/ApiMapReader.groovy:31]: prs.privateProperties != prs.customProperties

File: app/src/groovy/de/laser/api/v0/ApiWriterHelper.groovy (hunk 1 above)
Comment (croqueGrec09) [reply app/src/groovy/de/laser/api/v0/ApiWriterHelper.groovy:100]: person.privateProperties has been transposed into customProperties, customProperties does not exist, display is regulated by tenant settings and isPublic

File: app/src/groovy/de/laser/api/v0/ApiMapReader.groovy (hunk 2 above)
Comment (croqueGrec09) [reply app/src/groovy/de/laser/api/v0/ApiMapReader.groovy:31]: same as above

File: app/grails-app/domain/com/k_int/kbplus/Person.groovy [hunk 3]
```diff
@@ -53,7 +53,7 @@ class Person extends AbstractBaseWithCalculatedLastUpdated {
         roleLinks           cascade: 'all', batchSize: 10
         addresses           cascade: 'all', lazy: false
         contacts            cascade: 'all', lazy: false
-        privateProperties   cascade: 'all', batchSize: 10
+        customProperties   cascade: 'all', batchSize: 10
```
Comment (Moetez16) [inline app/grails-app/domain/com/k_int/kbplus/Person.groovy:56]: Die Bezeichnung verwirret sehr so. Vielleicht andere Bezeichnung?

Comment (croqueGrec09) [reply app/grails-app/domain/com/k_int/kbplus/Person.groovy:56]: Wieso verwirrt sie? Dann ist es einheitlich.

**Pull request merged**